reference-files/token-map.json

```diff
@@ -0,0 +1,5 @@
+{
+	"--primary-700": "--wp--custom--text--primary",
+	"--accent1-500": "--wp--custom--text--secondary",
```

Comment:
Picking these values based on looking at the Design Tokens defined in [here](https://designsystem.webstandards.ca.gov/style/tokens/#color-tokens). The idea is to take a css variable and map it to its corresponding theme.json token.